student-information-management-application/src/main/java/com/student/information/management/add/data/student/dao/impl/StudentDaoImpl.java

```diff
@@ -0,0 +1,98 @@
+package com.student.information.management.add.data.student.dao.impl;
+
+import com.student.information.management.add.app.model.student.Student;
+import com.student.information.management.add.data.connection.ConnectionHelper;
+import com.student.information.management.add.data.student.dao.StudentDao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.student.information.management.add.app.model.student.Student.isValidStudentId;
+import static com.student.information.management.add.data.utils.QueryConstant.*;
+/**
+ * This is the Student Dao Impl.
+ */
+public class StudentDaoImpl implements StudentDao {
+    Connection con = ConnectionHelper.getConnection();
+
+    @Override
+    public List<Student> getAllStudents() {
+        try {
```

Comment:
there is no return value when exception happens

Reply:
All Done po.